src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -514,6 +515,26 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 		return overrides;
 	}
 
+	private registerDefaultOverride(): void {
+		this._register(this.editorOverrideService.registerContributionPoint(
+			'*',
+			priorityToRank(ContributedEditorPriority.builtin),
+			{
+				id: DEFAULT_EDITOR_ASSOCIATION.id,
+				label: DEFAULT_EDITOR_ASSOCIATION.displayName,
+				detail: DEFAULT_EDITOR_ASSOCIATION.providerDisplayName,
+				active: (currentEditor) => this.fileEditorInputFactory.isFileEditorInput(currentEditor) && isEqual(currentEditor.resource, this.activeEditor?.resource),
```

Comment:
Do we need to account for `SideBySideEditor` too where we want to probe on the right hand side?

Reply:
The override service handles this, but I forgot to make sure we could also override the group so I've fixed that.